plugins/services/src/js/utils/DeclinedOffersUtil.js

```diff
@@ -201,14 +201,14 @@ const DeclinedOffersUtil = {
         unmatchedResource: declinedOffer.reason,
         offered: resources.reduce(
           (accumulator, resource) => {
-            const { name, role } = resource;
+            const { name, role, scalar = 0 } = resource;
```

Comment:
Good call! And to be honest, I don't know. I'll adjust this to make sure `null` values are replaced with `0` as well.